configure

```diff
@@ -797,6 +798,25 @@ read_registry_file()
 	done
 }
 
+substitute_words()
+{
+	local word new_words list
+	
+	word=$1
+	new_words=$2
+	list=$3
```

Comment:
Should these three lines have double quotes?

Reply:
You're right, it's better to use double quotes. thanks.